src/adservice/Dockerfile

```diff
@@ -29,8 +29,16 @@ ARG version=1.19.1
 WORKDIR /usr/src/app/
 
 COPY --from=builder /usr/src/app/ ./
-ADD https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/download/v$version/opentelemetry-javaagent.jar /app/opentelemetry-javaagent.jar
+ADD https://search.maven.org/remotecontent?filepath=io/sentry/sentry-opentelemetry-agent/6.9.1/sentry-opentelemetry-agent-6.9.1.jar /app/opentelemetry-javaagent.jar
+
 ENV JAVA_TOOL_OPTIONS=-javaagent:/app/opentelemetry-javaagent.jar
+ENV OTEL_TRACES_EXPORTER=otlp
+ENV OTEL_METRICS_EXPORTER=otlp
+ENV SENTRY_TRACES_SAMPLE_RATE=1.0
+ENV SENTRY_DEBUG=true
+
+#NOTE: The Sentry DSN will be injected into the container at runtime
+#ENV SENTRY_DSN=...
```

Comment:
To make it easier to run with docker-compose, we can actually leave `- SENTRY_DSN=${SENTRY_OTEL_DEMO_ADSERVICE_DSN}` environment variable line in `docker-compose.yml`, so that it can be passed by users more easily.

We can do it for all enabled services in a separate PR, though.

Reply:
Doing it for all services sounds like the way to go. Let's do it in a separate PR then.